### ISSUE TITLE
At 200% resize some of the data and controls present in "Extension details" section are getting truncated: A11y_VisualStudioCodeClient_UserWantsToExplore Customizations and Adding extension_UserWantsToReading details on specific extension

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#287226]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

### GitHub Tags: 
#A11yTCS;#Visual Studio Code Client;#E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2;#A11yMAS;#WCAG1.4.4; #ResizeText;

### Prerequisites:
1. Set the display resolution to 1366*768
2. Set the text size at 200

### Environmental Details: 
App Name: Visual Studio Code Client 
#URL: Visual Studio Code Insider
Microsoft Windows 11 Enterprise 
OS Build:  26200.7392

### Repro Steps: 
1.Open Visual Studio Code Insiders editor
2.Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3.Using Tab key navigate to the "More" option and select different options ("Show installed Extension, Show outdated extension, Show popular Extension,")
4.Click on any installed extension
5.TAB till "Extension details" section.
6. Observe the issue whether At 200% resize some of the data and controls present in "Extension details" section are getting truncated or not.

### Actual Results: 
At 200% resize some of the data and controls present in "Extension details" section are getting truncated 

### Expected Results: 
At 200% resize also all the data and controls present in "Extension details" section should not be getting truncated and should be properly visible.

### User Impact: 
Low vision user when zoom the screen at 200 for better visibility will not be able to see complete data and controls present in "Extension details" section.

### Attachment:

<img width="1365" height="717" alt="Image" src="https://github.com/user-attachments/assets/0f64c981-adf7-4449-b29f-bece07d6dd9a" />
<img width="1365" height="714" alt="Image" src="https://github.com/user-attachments/assets/6f7f65c4-d2cd-4476-8923-66f009cbaf6f" />
